src/os/bluestore/BlueStore.cc

```diff
@@ -6147,6 +6150,142 @@ void BlueStore::_wctx_finish(
   o->onode.compress_extent_map();
 }
 
+
+bool BlueStore::blobs_need_garbage_collection(
+  OnodeRef o,
+  uint64_t start_offset, 
+  uint64_t end_offset,
+  uint32_t *blob_depth,
+  uint64_t *gc_start_offset, 
+  uint64_t *gc_end_offset)
+{
+  uint32_t depth = 0;
+  bool head_overlap = false;
+  bool tail_overlap = false;
+
+#ifdef DISPLAY_EXTENT_DEPTH
+  {
+    map<uint64_t,bluestore_lextent_t>::iterator bp =
+	  o->onode.extent_map.begin();
+    while(bp != o->onode.extent_map.end()) {
+        derr << " Extent Start = " << bp->first << " Extent Length = " << bp->second.length
+             << " Depth = " << bp->second.blob_depth << dendl;
+        ++bp;
+    }
+  }
+#endif
```

Comment:
@varadakari Why within #ifdef ? Also why derr ? 
Need to do it with say dout (40)